CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed
+
+- Migrate filelog operators to follow v0.29.0 changes (#436)
+  - [BREAKING CHANGE] OpenTelemetry Logs collection has updated field syntax. Field syntax
+    no longer requires the $ character. Instead each field must begin with body,
+    attributes, or resource. Also they have updated/removed some of the operators.
+    If you are using `extraFileLogs` or `logsCollection.containers.extraOperators` config,
+    see [Upgrading Guideline](https://github.com/open-telemetry/opentelemetry-log-collection/blob/v0.29.0/CHANGELOG.md#upgrading-to-v0290)
+
```

Comment:
A lot of this looks good, I'd suggest making these updates. We should also update the title of this PR to match this (include  opentelemetry-log-collection v0.29.0).

- Migrate filelog operators to follow opentelemetry-log-collection v0.29.0 changes (#436)
  - [BREAKING CHANGE] Several breaking changes were made that affect the
    filelog, syslog, tcplog, udplog, and journald receivers. Any use of the
    [extraFileLogs](https://github.com/signalfx/splunk-otel-collector-chart/blob/941ad7f255cce585f4c06dd46c0cd63ef57d9903/helm-charts/splunk-otel-collector/values.yaml#L488)
    config,
    [logsCollection.containers.extraOperators](https://github.com/signalfx/splunk-otel-collector-chart/blob/941ad7f255cce585f4c06dd46c0cd63ef57d9903/helm-charts/splunk-otel-collector/values.yaml#L431)
    config, and affected receivers in a custom manner should be reviewed. See
    [upgrading guidelines](https://github.com/open-telemetry/opentelemetry-log-collection/blob/v0.29.0/CHANGELOG.md#upgrading-to-v0290)

Reply:
Today my team merged in the code for the splunk-otel-collector v0.49.0 release. Let's try to update this, I'll approve it, and I'll merge it.

Reply:
Today my team merged in the code for the splunk-otel-collector v0.49.0 release. Let's try to update this, I'll approve it, and I'll merge it.